portal/static/portal/js/common.js

```diff
@@ -87,6 +87,12 @@ function openConfirmationBox(name) {
     $('#confirmation-dialog').dialog('open');
 }
 
+function postWithCsrf(path) {
```

Comment:
Could we pass a Csrf param into `params` of `post` instead of making a new function? Any reason against?

Reply:
nope we can do this - i was starting with a "soft refactoring" :joy: 
I'll have a look

Reply:
I think I actually like that function as a "shortcut" so we don't have to paste the token everywhere (see next comment)

Reply:
Sure 👍

Reply:
This is not reviewable, I can't resolve this discussion = (